src/ActiveLogin.Identity.Swedish.TestData/TestData.fs

```diff
@@ -0,0 +1,49 @@
+namespace ActiveLogin.Identity.Swedish.FSharp.TestData
+open ActiveLogin.Identity.Swedish.FSharp
+open ActiveLogin.Identity.Swedish.TestData.AllPins
+open System
+
+
+module SwedishPersonalIdentityNumberTestData =
+    let private rng = Random()
+    let private random _ = rng.Next()
+    let internal shuffledPins() = allPins |> Array.sortBy random
+    let raw12DigitStrings= 
+        allPins
+        |> Array.map (fun (year, month, day, birthNumber, checksum) -> sprintf "%i%i%i%i%i" year month day birthNumber checksum)
+
+    let internal create (year, month, day, birthNumber, checksum) =
+        let values =
+            { Year = year
+              Month = month
+              Day = day
+              BirthNumber = birthNumber
+              Checksum = checksum }
+        match SwedishPersonalIdentityNumber.create values with
+        | Ok p -> p
+        | Error _ -> failwith "broken test data" 
+
+    let allPinsSorted() = seq { for pin in allPins do yield create pin }
```

Comment:
When doing the XML comments, please define how they are sorted, or maybe even suffix the funciton name with it. I'd guess they are sorted by date, desc?